src/libponyc/codegen/genexe.c

```diff
@@ -277,6 +278,9 @@ static bool link_exe(compile_t* c, ast_t* program,
 #ifdef PONY_USE_LTO
     "-flto -fuse-linker-plugin "
 #endif
+#if defined(PLATFORM_IS_ALPINE_LINUX) || defined(PLATFORM_IS_FREEBSD)
```

Comment:
@jemc This is unlikely to be correct. See the note in my main PR comment about `libexecinfo`/`execinfo.h`. Additionally, there is no way to detect that the libc being used is `musl` (see: http://wiki.musl-libc.org/wiki/FAQ#Q:_why_is_there_no_MUSL_macro_.3F).

The way this is currently detected is via a `Makefile` check for the existence of `/etc/alpine-release` and is tied to the Alpine Linux distribution.